python/pyproject.toml

```diff
@@ -47,3 +47,13 @@ ignore_missing_imports = true
 
 [tool.coverage.run]
 source = ['src/']
+
+[tool.taskipy.tasks]
+install = "pip install -e \".[dev,arrow]\""
+lint = "pre-commit run --all-files"
+test = """
+    coverage run --source=src/ -m pytest tests/ &&
```

Comment:
I would suggest getting rid of the indentation here, probably this will give us a nicer listing:

Before:
```
$ task -l
install pip install -e ".[dev,arrow]"
lint    pre-commit run --all-files
test        coverage run --source=src/ -m pytest tests/ &&     coverage report -m --fail-under=90 &&     coverage html &&     coverage xml
```

after:
```
➜  python git:(taskipy) ✗ task -l             
install pip install -e ".[dev,arrow]"
lint    pre-commit run --all-files
test    coverage run --source=src/ -m pytest tests/ && coverage report -m --fail-under=90 && coverage html && coverage xml
```